src/api/customer/customer-business.ts

```diff
@@ -11,6 +11,34 @@ const persistCustomer: CustomerCreationRequestHandler = async (req, res, next) =
     }
 }
 
+const persistFindAllCustomers: CustomersFindAllRequestHandler = async (req, res, next) => {
+    
+    try {       
+        const off = req.query.offset
+        const lim = req.query.limit
+        const users:Customer[] = await Customer.findAll({limit: parseInt(lim) || undefined, offset: parseInt(off) || undefined});
```

Comment:
Conforme comentado acima, essa tratativa do `limit` e `offset` poderia ser feito direto no Joi.

Porém, achei verboso o código da linha 19, sem contar que a tipagem `Customers[]` é desnecessária.

A própria inferência de tipo do typescript já sabe que a resposta do método `Customer.findAll(...)` retorna uma promisse de um array de Customer `Promise<Customer[]>`

Que tal assim:
```suggestion
        const findAllOptions = {
            offset: parseInt(req.query.offset) || undefined,
            limit: parseInt(req.query.limit) || undefined
        }
        const users = await Customer.findAll(findAllOptions);
```